itests/test-jdbc/src/test/java/org/apache/hive/jdbc/TestStandaloneJdbc.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hive.jdbc;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.lang.reflect.Field;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.hadoop.hive.metastore.utils.SecurityUtils;
+import org.apache.hadoop.security.UserGroupInformation;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class TestStandaloneJdbc {
+  private static File workDir = new File(System.getProperty("java.io.tmpdir"), "test-it-standalone-jdbc");
+  private static ITAbstractContainer HS2;
+  private static ITAbstractContainer ZK_HS2;
+
+  @BeforeClass
+  public static void setup() throws Exception {
+    workDir.mkdirs();
+    HS2 = new ITHiveServer2(workDir);
+    ZK_HS2 = new ITZKHiveServer2(workDir);
+
+    HS2.start();
+    ZK_HS2.start();
+  }
+
+  @Test
+  public void testBinaryJdbc() throws Exception {
+    testMetaOp(HS2.getBaseJdbcUrl(), true);
```

Comment:
I don't think if HS2 as a parameter is necessary: it is a private static variable of this test class.